src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -10,47 +12,6 @@ public static class ExpectedExceptionTests
 {
     [Fact]
     [OuterLoop]
-    public static void NotExistentHost_Throws_EndpointNotFoundException()
```

Comment:
I don't really understand this test - it seems to do exactly what `UnknownUrl_Throws_EndpointNotFoundException` does, except with the funky while loop that doesn't make sense to me. Is there something I'm missing? I ended up amalgamating the two tests into one [Theory] based test. 

One endpoint is for a *host* that doesn't exist, one is for an *endpoint* that doesn't exist on a host that does.